src/goods.ts

```diff
@@ -30,11 +32,14 @@ export { default as path } from 'node:path'
 export { default as os } from 'node:os'
 export { ssh } from 'webpod'
 
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = basename(__filename)
 export let argv = minimist(process.argv.slice(2))
+argv._ = argv._.filter(arg => arg !== __dirname)
```

Comment:
This is a wrong solution. Instead of trying to fix consequences, we need to fix root cause: slice argv to a correct amount.

Reply:
Thanks for the heads up! I was actually running scripts using node, hence the oversight.